ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### Flow
```

Comment:
I think this section should be called 'Prerequisites' or 'First Steps', or something.